umm-spec-lib/resources/json-schemas/variable/umm/v1.9/umm-var-search-results-json-schema.json

```diff
@@ -0,0 +1,286 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+    "ItemType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "Represents a single item found in search results. It contains some metadata about the item found along with the UMM representing the item. umm won't be present if the item represents a tombstone.",
```

Comment:
nitpik: UMM and umm in the same description, pick one (UMM)

Reply:
I didn't change this file. I just brought it forward. Looking at the code base, this was in the initial release 6 years ago.  I updated the text.